dataset_builders/pie/aae2/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pie-datasets>=0.6.0,<0.9.0
+pie-modules>=0.8.0,<0.9.0
```

Comment:
I think we require `0.8.3` as min version for `pie-modules` (because of strict tokenization with partitions, see below)

Reply:
edited in 491c137